packages/field-base/src/aria-label-mixin.js

```diff
@@ -9,27 +9,14 @@ import { InputMixin } from './input-mixin.js';
 
 const AriaLabelMixinImplementation = (superclass) =>
   class AriaLabelMixinClass extends InputMixin(LabelMixin(superclass)) {
-    constructor() {
-      super();
-
-      this.__preventDuplicateLabelClick = this.__preventDuplicateLabelClick.bind(this);
-    }
-
-    /** @protected */
-    connectedCallback() {
-      super.connectedCallback();
-
-      if (this._labelNode) {
-        this._labelNode.addEventListener('click', this.__preventDuplicateLabelClick);
-      }
-    }
-
     /** @protected */
-    disconnectedCallback() {
-      super.disconnectedCallback();
+    ready() {
```

Comment:
**note:** We should prefer the `ready` callback over `connectedCallback` for the logic which accesses internal elements.

Once migrating to Lit, `ready` becomes `firstUpdated`, but `connectedCallback` works just the old way which means if you still access internal elements in `connectedCallback`, they will not be available (since they are not yet rendered at the moment of `connectedCallback`).